Dockerfile

```diff
@@ -62,36 +73,41 @@ ENV PYENV_ROOT /home/docs/.pyenv
 ENV PATH /home/docs/.pyenv/shims:$PATH:/home/docs/.pyenv/bin
 
 # Install supported Python versions
-RUN pyenv install 2.7.13 && \
-    pyenv install 3.6.2 && \
-    pyenv install 3.5.4 && \
-    pyenv install 3.4.7 && \
-    pyenv install 3.3.6 && \
-    pyenv global 2.7.13 3.6.2 3.5.4 3.4.7 3.3.6
+RUN pyenv install $PYTHON_VERSION_27 && \
+    pyenv install $PYTHON_VERSION_36 && \
+    pyenv install $PYTHON_VERSION_35 && \
+    pyenv install $PYTHON_VERSION_34 && \
+    pyenv install $PYTHON_VERSION_33 && \
+    pyenv global \
+        $PYTHON_VERSION_27 \
+        $PYTHON_VERSION_36 \
+        $PYTHON_VERSION_35 \
+        $PYTHON_VERSION_34 \
+        $PYTHON_VERSION_33
 
 WORKDIR /tmp
 
-RUN pyenv local 2.7.13 && \
+RUN pyenv local $PYTHON_VERSION_27 && \
     pyenv exec pip install -U pip && \
     pyenv exec pip install --only-binary numpy,scipy numpy scipy && \
     pyenv exec pip install pandas matplotlib virtualenv
 
-RUN pyenv local 3.6.2 && \
+RUN pyenv local $PYTHON_VERSION_36 && \
     pyenv exec pip install -U pip && \
     pyenv exec pip install --only-binary numpy,scipy numpy scipy && \
     pyenv exec pip install pandas matplotlib virtualenv
 
-RUN pyenv local 3.5.4 && \
+RUN pyenv local $PYTHON_VERSION_35 && \
     pyenv exec pip install -U pip && \
     pyenv exec pip install --only-binary numpy,scipy numpy scipy && \
     pyenv exec pip install pandas matplotlib virtualenv
 
-RUN pyenv local 3.4.7 && \
+RUN pyenv local $PYTHON_VERSION_34 && \
```

Comment:
I thought we removed 34?

Reply:
I think it's a mistake, since he said it doesn't build because of an incompatibility issue. So, this will fail. I suppose this whole block has to be removed.

Reply:
hrm yeah, might have been a mix up because i did a separate pr that just made the label changes first. I thought of a better way to track changes to the build image, just tracking the image sha, so we can probably merge the two PRs here into one really.